docker/_Dockerfile

```diff
@@ -19,4 +19,6 @@ RUN export uid={{ uid }} gid={{ gid }} && \
 RUN groupadd docker -g {{ docker_gid }} && \
 	gpasswd -a {{ internal_user }} docker
 
-ENV HOME /home/{{ internal_user }}
\ No newline at end of file
+ENV HOME /home/{{ internal_user }}
```

Comment:
I agree, it's better!

However, I'd rather one-line the 3 ENV commands with 
```
ENV HOME=/home/{{ internal_user }} \
    LANGUAGE=en \
    LANG={{ default_locale }}
```
So there is only one layer created for 3 env variables.